src/api/protectedApiClient.ts

```diff
@@ -52,11 +57,22 @@ const makePFRequest = (): Promise<void> => {
     .catch((err) => err);
 };
 
+const getEventAnnouncements = (
+  eventId: number,
+): Promise<EventAnnouncement[]> => {
+  return AppAxiosInstance.get(
+    `${ProtectedApiClientRoutes.ANNOUNCEMENTS}/${eventId}`,
+  )
+    .then((res) => res.data?.announcements)
+    .catch((err) => err);
```

Comment:
Have you tested if there is an error fetching announcements? Do we want to catch the error here?

Reply:
I'm still pretty sure we do. .catch here will catch specifically the AxiosError, then return a rejected promise that the thunk will deal with.